conda/base/constants.py

```diff
@@ -96,6 +96,15 @@
 UNKNOWN_CHANNEL = "<unknown>"
 
 
+class SafetyChecks(Enum):
+    disabled = 'disabled'
+    warn = 'warn'
+    enabled = 'enabled'
+
+    def __str__(self):
+        return self.value
```

Comment:
where is this `self.value` created, should not we add something on the `__init__` ??